src/docs/product/integrations/github/index.mdx

```diff
@@ -220,6 +220,30 @@ You can also resolve issues with pull requests by including `fixes <SENTRY-SHORT
 
 When Sentry sees this, we’ll automatically annotate the matching issue with a reference to the commit or pull request, and, later, when that commit or pull request is part of a release, we’ll mark the issue as resolved.
 
+### Stack Trace Linking
+Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. Otherwise we'll link you to the current state of the source code (using the `default_branch`).
+
+1. Navigate to **Settings > Integrations > GitHub > Configurations**.
+
+1. Next to your GitHub Instance, click "Configure".
```

Comment:
Start steps with an action, like this (though I am uncertain if this is a button):
```suggestion
1. Click the "Configure" button next to your GitHub Instance.
```